apps/notifications/src/panel/templates/comment-reply-input.jsx

```diff
@@ -144,6 +144,7 @@ class CommentReplyInput extends Component {
 
 		if ( event ) {
 			event.preventDefault();
+			event.stopPropagation();
```

Comment:
Prevent the notification panel from closing because of the clicking outside behavior (the SEND button changed to the loading spinner so the target element was not inside the panel anymore)